proto/gen_proto.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# pinned plugin versions
+PLUGIN_GO_VERSION="v1.31.0"
+PLUGIN_GO_GRPC_VERSION="v1.3.0"
+PLUGIN_GRPC_GATEWAY_VERSION="v2.18.0"
+
+# crete the build directory (.build) to store the generated binaries
+BUILD_DIR="$(pwd)/.build"
+mkdir -p "${BUILD_DIR}/bin"
+
+# export the binary path of the go plugins
+export GOBIN="${BUILD_DIR}/bin"
```

Comment:
please do not export vars to not change user environment.
try the following approach: `GOBIN="${GOBIN}" go install...`